src/webots/core/WbPreferences.cpp

```diff
@@ -28,6 +28,7 @@
 #include <QtCore/QRegularExpressionMatch>
 #include <QtCore/QStandardPaths>
 
+#include <iostream>
```

Comment:
Is this include really needed?
Because it doesn't seem that related changes have been added.